rls-quick-start-guide.md

```diff
@@ -0,0 +1,72 @@
+# RLS Quick-Start Guide
+
+Hello, RLS members new and old! Welcome to the Rands Leadership Slack quick-start guide 🙂 
+
+RLS is a big community! The last time Rands made an #announcement, RLS membership was the same size as East Peoria, Illinois with 22,620 people. As RLS continues to grow, it can be harder to figure out where to get started. This quick-start guide is intended to give you that starting point to exploring and participating in this community. 
```

Comment:
Unless the intent is to update this document after every announcement, referencing the current date (in a non-ambiguous date format!) might be valuable.

eg.  "Rand's July 2022 post in #announcement showed the membership was the same size…"

Reply:
Oh good call! TY!